pushnotifications/src/test/java/com/pusher/pushnotifications/internal/DeviceStateStoreTest.kt

```diff
@@ -11,37 +11,17 @@ import org.mockito.Mockito.`when`
 import org.junit.Before
 import org.mockito.Mockito.doNothing
 
-class DeviceStateStoreTest {
+class InstanceDeviceStateStoreTest {
   private val context: Context = Mockito.mock(Context::class.java)
   private val sharedPrefs = Mockito.mock(SharedPreferences::class.java)
-  private lateinit var testDeviceStateStore: DeviceStateStore
+  private lateinit var testInstanceDeviceStateStore: InstanceDeviceStateStore
 
   @Before
   fun before() {
-    `when`(context.getSharedPreferences("com.pusher.pushnotifications.PushNotificationsInstance", Context.MODE_PRIVATE))
+    `when`(context.getSharedPreferences("com.pusher.pushnotifications.i-123.PushNotificationsInstance", Context.MODE_PRIVATE))
         .thenReturn(sharedPrefs)
 
-    this.testDeviceStateStore = DeviceStateStore(context)
-  }
-
-  @Test
-  fun `instanceId is retrieved correctly`() {
```

Comment:
do we still need a "higher" test to ensure that the instanceId section is saved correctly?